stubs/DummyViewModel.stub

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace DummyNamespace;
+
+use Spatie\ViewModels\ViewModel;
+
+class DummyClass extends ViewModel
```

Comment:
Could you add an empty constructor to this class?

Reply:
done. hope you wanted the comment line ready to position the cursor/caret